doc/changelog/11-infrastructure-and-dependencies/10471-ocaml-408.rst

```diff
@@ -0,0 +1,5 @@
+- Coq now officially supports OCaml 4.08.
+
+  see :ref:`ocaml+408` for details
```

Comment:
Hum, what's this?

Reply:
No idea, I copied it from other entries.

Reply:
Where did you copy `` :ref:`ocaml+408` `` from? grep doesn't find it.

Reply:
Ok let me see this.

Reply:
Oh indeed, so that should be a reference for the install section?

Reply:
What's the label there? It seems we don't have a section for install / requirements in the manual.

Reply:
Indeed, we don't. At the moment, you should refer to the `INSTALL` file instead.
FWIW with have plans with Matthieu and Jim to refactor the manual structure (CEP incoming) and that should allow to add such sections on install / requirements, and also on getting started like you once suggested.